docs/release-notes.md

```diff
@@ -1,5 +1,36 @@
 # Release notes
 
+## Trino Gateway 10 (?? July 2024)
+
+[JAR file gateway-ha-10-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/10/gateway-ha-10-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:10`
+
+* [:warning: Breaking change:](#breaking) Remove support for any Dropwizard and
+  Jetty Proxy integration and usage. Add
+  [Airlift](https://github.com/airlift/airlift) as base application framework as
+  used in Trino. This changes the supported Trino Gateway startup, configuration
+  files, and relevant properties. Find details in the documentation, and
+  specifically refer to the [upgrade guide](migration-to-airlift.md) when
+  migrating from older releases.
+  ([#41](https://github.com/trinodb/trino-gateway/issues/41))
+* [:warning: Breaking change:](#breaking) Improve Helm chart reliability and
```

Comment:
"adjust to new Airlift base framework" doesn't feel like it means much from a user-facing perspective, why is this a breaking change?

Reply:
because any custom modules are breaking and any existing config that relies on dropwizard (and there is a whole bunch) also breaks.